cmd/mavenBuild.go

```diff
@@ -129,10 +129,12 @@ func runMavenBuild(config *mavenBuildOptions, _ *telemetry.CustomData, utils mav
 
 	goals = append(goals, "org.jacoco:jacoco-maven-plugin:prepare-agent")
 
-	if config.Verify {
-		goals = append(goals, "verify")
-	} else {
-		goals = append(goals, "install")
+	if !config.Publish {
```

Comment:
during the publish phase the install is not run again isnt it ? , its already taken care via https://pages.github.tools.sap/project-piper/steps/mavenBuild/#deployflags